apps/services/models.py

```diff
@@ -183,13 +189,13 @@ def __repr__(self):
                                   Column('room_number', INTEGER),
                                   Column('bed_place_name', VARCHAR(50)),
                                   Column('month_price', DECIMAL(6, 2)),
-                                  Column('start_date_accommodation', TIMESTAMP),
-                                  Column('end_date_accommodation', TIMESTAMP),
+                                  Column('start_accommodation_date', TIMESTAMP),
```

Comment:
strange, why not date type for both columns?

Reply:
Left it as it was before, and now it's working because column type is defined in main.sql. But of course the problem will raise after migrations appears. I will change it.